.github/workflows/circuit-build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14, 16, 18]
+        node-version: [18]
```

Comment:
So with this change, should we be using/recommending Node 18 across the board?

If so, we should update README & docs which instruct Node 14/16:
- https://github.com/privacy-scaling-explorations/maci#requirements
- https://privacy-scaling-explorations.github.io/maci/installation.html#requirements

Reply:
Yes, I can add the updates on this PR.

Reply:
Thanks!